rskj-core/src/main/java/org/ethereum/net/rlpx/HandshakeHandler.java

```diff
@@ -377,13 +377,15 @@ private void processHelloMessage(ChannelHandlerContext ctx, HelloMessage helloMe
         List<Capability> capInCommon = configCapabilities.getSupportedCapabilities(helloMessage);
         channel.initMessageCodes(capInCommon);
         for (Capability capability : capInCommon) {
-            // It seems that the only supported capability is RSK, and everything else is ignored.
+            // Accept RSK capability.
```

Comment:
I would say, RSK is the condition needed to be able to finish the Handshake successfully.

Reply:
solved